addons/sourcemod/scripting/prophunt.sp

```diff
@@ -244,7 +242,9 @@ public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max
 public void OnMapStart()
 {
 	g_IsMapRunning = true;
-	
+
```

Comment:
Please add a tab on the two empty lines to be consistent with the rest of the file